html-fundamentals/content/nutrition.html

```diff
@@ -6,6 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Wired Brain Coffee - Nutrition Information</title>
+    <!--<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">-->
+    <link rel="stylesheet" href="../styles.css">
+
```

Comment:
why keep this in?

Reply:
I might add it in later to get better style across the whole file rather than build it in the styles.css file myself